dev/plugins/hu.elte.txtuml.api.model/src/hu/elte/txtuml/api/model/Action.java

```diff
@@ -50,6 +50,7 @@
 	 * @throws NullPointerException
 	 *             if <code>classType</code> is <code>null</code>
 	 */
+	@ExternalBody
```

Comment:
@kovacsgabor, Why do we have `@ExternalBody` here? This code is not part of a model.